Tests/AsyncHTTPClientTests/HTTPClientTests.swift

```diff
@@ -2533,4 +2533,59 @@ class HTTPClientTests: XCTestCase {
         XCTAssertEqual(info.connectionNumber, 1)
         XCTAssertEqual(info.requestNumber, 1)
     }
+
+    func testBackpressure() {
```

Comment:
I think we should maybe name it `testDownloadBackpressure`? That'd match the [existing test](https://github.com/swift-server/async-http-client/blob/aac4357b655231e2d31a9e24fc98756f95c73fd5/Tests/AsyncHTTPClientTests/HTTPClientInternalTests.swift#L232).

Asl

Reply:
Ok